src/olympia/amo/tests/test_models.py

```diff
@@ -190,6 +190,7 @@ def test_filename(self):
         preview = self.get_object()
         assert 'png' in preview.thumbnail_path
         assert 'png' in preview.image_path
+        assert 'png' in preview.original_path
```

Comment:
nitpick, can we change these tests to `assert preview.thumbnail_path.endswith('.png')` to be a bit more failure-proof and make sure we actually have that as an ext of the filename.

Reply:
https://github.com/mozilla/addons-server/pull/8553/commits/d002da8dc8c33a60929d9ab9fec9643d928c28c2 but I went too far and had to fix with https://github.com/mozilla/addons-server/pull/8553/commits/a37b5c605d644b592578b097c783dc64c59b2a7c